agent/agent.go

```diff
@@ -860,12 +860,6 @@ func (a *Agent) Start(ctx context.Context) error {
 		go m.Monitor(&lib.StopChannelContext{StopCh: a.shutdownCh})
 	}
 
-	// consul version metric with labels
-	metrics.SetGaugeWithLabels([]string{"version"}, 1, []metrics.Label{
-		{Name: "version", Value: a.config.VersionWithMetadata()},
```

Comment:
Have the `emitVersionUsage()` emit the same value `VerstionWithMetadata()` instead of `GetHumanVersion()`?

Reply:
VersionWithMetadata looks like it just concats Version and VersionMetadata [here](https://github.com/hashicorp/consul/blob/255aa158db8ccbfd445205654e153742b127116d/agent/config/runtime.go#L1738), which get input from the version module [here](https://github.com/hashicorp/consul/blob/255aa158db8ccbfd445205654e153742b127116d/agent/config/default.go#L268), so this should be doable.